packages/lib/src/vis/shared/Overlay.tsx

```diff
@@ -0,0 +1,153 @@
+import { isDefined } from '@h5web/shared';
+import type { GroupProps } from '@react-three/fiber';
+import { useFrame, useThree } from '@react-three/fiber';
+import {
+  useState,
+  useLayoutEffect,
+  useRef,
+  useCallback,
+  useEffect,
+} from 'react';
+import type { HTMLAttributes, RefCallback } from 'react';
+import ReactDOM from 'react-dom';
+import type { Group } from 'three';
+import { Vector3 } from 'three';
+
+import { useAxisSystemContext } from './AxisSystemContext';
+
+interface Props extends HTMLAttributes<HTMLDivElement> {
+  refCallback?: RefCallback<HTMLDivElement>;
+  groupProps?: GroupProps;
+  followCamera?: boolean;
+  scaleOnZoom?: boolean;
+}
+
+// Customised version of drei's `<Html>` component
+// https://github.com/pmndrs/drei/blob/v6.0.3/src/web/Html.tsx
+function Overlay(props: Props) {
```

Comment:
I guess this is a copy of the old `Html` and that you will use the new `Html` in a future PR.

Reply:
Indeed.